projects/demo/src/pages/kit/date-time/date-time-mask-doc.template.html

```diff
@@ -63,6 +63,32 @@
             </ng-template>
             <date-time-mask-doc-example-3 />
         </tui-doc-example>
+
+        <tui-doc-example
+            id="time-step"
+            heading="Time segments stepping via arrows"
+            [content]="dateTimeTimeStepExample"
+            [description]="timeStepDescription"
+        >
+            <ng-template #timeStepDescription>
+                <p class="tui-space_top-0 tui-space_bottom-1">
+                    Property
+                    <code>timeStep</code>
+                    allows you to increment and decrement time segments by pressing
+                    <code>UpArrow</code>
+                    and
+                    <code>DownArrow</code>
```

Comment:
```suggestion
                    <code>ArrowUp</code>
                    and
                    <code>ArrowDown</code>
```

(the same naming as `event.key` of `keydown` event https://w3c.github.io/uievents/#events-keyboard-key-location)